README.md

```diff
@@ -1,24 +1,29 @@
-# README
+# Apply for postgraduate teacher training
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This service enables postgraduate candidates to apply for initial teacher
+training.
 
-Things you may want to cover:
+## Initial setup
 
-* Ruby version
+Clone this repo and run `bundle install`.
 
-* System dependencies
+## Running locally
 
-* Configuration
+```
+rails s
+```
 
-* Database creation
+## Running the tests and linter
 
-* Database initialization
+```
+bundle exec rake
+```
 
-* How to run the test suite
+## Date and time formats
```

Comment:
It's not clear to me what this is describing. Is it needed in the README?

Reply:
:+1: Let's remove it